src/components/common/Avatar/Avatar.tsx

```diff
@@ -12,27 +13,32 @@ import {
 } from './Avatar.style';
 
 interface AvatarProps extends React.HTMLAttributes<HTMLDivElement> {
-  avatarShape: AvatarShapeType['avatarShape'];
+  avatarSize: AvatarSizeType['avatarSize'];
```

Comment:
p2;
AvatarSizeType을 interface가 아닌 type으로 선언하면 좀 더 명확하게 표현할 수 있을것 같아요!
```suggestion
  avatarSize: AvatarSizeType;
```